### DISCUSSION TITLE
Welcome to the-coding-club Discussions!

### DISCUSSION THREAD

**Discussion by jinbow** [podaac/the-coding-club#13] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by yangleir** [comment]
Hi, Dr Wang
I am happy to be the first one to discuss something here. My name is Yang Lei. I am now a PhD student in Institute of Oceanology , Chinese Academy of Sciences. My main research  moves to the physical oceanography (also related the SWOT) recently. 

I have paid a lots of times to learn the usage of LLC4320 model. And not surprisingly, I found the `Pangeo Cloud` to be the best approach for me. After touched the Cloud computing, I feel this is very  fascinating. It can meet some of my requirement for using LLC4320 data. You must also familiar with the Pangeo group.

However, I have some questions about the Pangeo Cloud. And after post some topics in the Pangeo forum, I got no response.  Because the LLC4320 is highly related to the NASA SWOT group, I guess this maybe one place to discuss this.

- The LLC4320 on Pangeo Cloud only has the surface layer? Is that true? If so, is there one method to use the other 89 layers, especially using the Cloud approach. Maybe based on NASA PO.DAAC?
- Another issue is that the SSH data are  end at 2017 on Pangeo Cloud. Seems there is no plan to add the new dataset by the Pangeo group. So this is not good the related applications because it already missing 5 years data . I just wonder is there any plan for NASA to make the fully SSH data on Cloud and make it easy to use?

Thanks

Lei
